google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -2330,56 +2200,6 @@ public void test_hql_group_by_example_2() {
 		});
 	}
 
-	@Test
-	@RequiresDialect(H2Dialect.class)
-	@RequiresDialect(PostgreSQL81Dialect.class)
-	@RequiresDialect(MySQL5Dialect.class)
-	public void test_hql_group_by_example_3() {
```

Comment:
Would this one also work in Spanner? I know this one specifies it only works for some dialects, but it mentions both postgres and mysql, which feel like pretty general SQL to me

Reply:
Hmm it didn't work; seemed to fail at the group by. This one looks like it might be able to make work with some adjustments to Dialect; created issue to track: https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/issues/56